app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class UserMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def welcome_email
+    @user = params[:user]
```

Comment:
Sao lại bỏ `@url` rồi em

Reply:
Đọc cái này ròi làm nha em https://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views
Nhờ config cho cả development và production